adafruit_drv2605.py

```diff
@@ -148,7 +148,9 @@ def stop(self):
 
     @property
     def mode(self):
-        """Get and set the mode of the chip.  Should be a value of:
+        """
+        Get and set the mode of the chip.  Should be a value of:
```

Comment:
This should read `The mode of the chip. Should be a value of:`. Avoid use of "get" and "set" in documentation.